graph/db/kv_store.go

```diff
@@ -4663,6 +4684,22 @@ func deserializeChanEdgePolicyRaw(r io.Reader) (*models.ChannelEdgePolicy,
 		edge.ExtraOpaqueData = opq[8:]
 	}
 
+	// Attempt to extract the inbound fee from the opaque data. If we fail
+	// to parse the TLV here, we return an error we also return the edge
+	// so that the caller can still use it. This is for backwards
+	// compatibility in case we have already persisted some policies that
+	// have invalid TLV data.
+	var inboundFee lnwire.Fee
+	typeMap, err := edge.ExtraOpaqueData.ExtractRecords(&inboundFee)
+	if err != nil {
+		return edge, fmt.Errorf("%w: %w", ErrParsingExtraTLVBytes, err)
```

Comment:
Q: do we plan to do anything with this error? Right now we return it and then match everywhere if it's `ErrParsingExtraTLVBytes` to ignore it. Maybe we could use the same approach we already do in rpcserver.go's `extractInboundFeeSafe(data lnwire.ExtraOpaqueData) lnwire.Fee` and ignore it right here at parsing time? wdyt?

Reply:
(gonna request override merge in the mean time - happy to address action items from this discussion in a follow up)